src/components/datagrid/datagrid.js

```diff
@@ -6601,6 +6601,11 @@ Datagrid.prototype = {
       });
     }
 
+    if (this.settings.toolbar.title && $('.lookup-modal .modal-title')) {
+      const optionTitle = this.settings.toolbar.title;
```

Comment:
```suggestion
      const optionTitle = this.settings.toolbar.title;
```
This looks good but a couple concerns.

1) how can we guarantee this wont effect other lookups in the page? We might need to find a way to populate this within the lookup formatter to be sure. I think its worth adding another lookup on a button the test page to test
2) Can we default the title to the column name if no toolbar title is used? 
3) i think the title should be on the column as this is the general grid tooltbar title i.e. `editorOptions: {title: 'Products'}` (default to column name otherwise)
4) slight layout issue on the results text

Reply:
Should we use the title outside of the modal instead the column name? I just noticed that in the lookup, it uses it if there's no title in the formatter.

Reply:
What do you mean by title outside the modal? Was thinking it would use either the editorOptions: {title: 'something'} on the column or the column "name" if thats not there.

Its likely going to be selecting values for that column so its tied to the column data not the grid title. If that answers the question?

Reply:
Gotcha. Yeah, I couldn't get the value of the grids title either.